lxd/api_internal.go

```diff
@@ -424,7 +426,7 @@ func internalSQLGet(d *Daemon, r *http.Request) response.Response {
 		return response.SmartError(errors.Wrap(err, "failed to start transaction"))
 	}
 	defer tx.Rollback()
-	dump, err := query.Dump(tx, schema, schemaOnly == 1)
+	dump, err := query.Dump(ctx, tx, schema, schemaOnly == 1)
```

Comment:
The `tx` created above should use `db.BeginTx()` which can accept the context too.